common/hashes/hashes.go

```diff
@@ -62,3 +65,21 @@ func Simhash(data []byte) string {
 	hash := simhash.Simhash(simhash.NewWordFeatureSet(data))
 	return fmt.Sprintf("%d", hash)
 }
+
+func Jarm(host string) string {
+	t := target{}
+	if u, err := url.Parse(host); err == nil {
```

Comment:
I think we can skip the check if the url starts with `http` and return `""` directly (no tls involved)